docs/source/install.rst

```diff
@@ -49,13 +51,17 @@ Download and install miniconda_. Now run the following commands:
     cd Autometa
     # Construct the autometa environment from requirements.txt
     conda create -n autometa -c conda-forge -c bioconda --file=requirements.txt
+    # Activate environment
+    conda activate autometa
+    # install trimap for kmer embedding
+    pip install trimap
```

Comment:
`pip install densmap-learn` should also be added.

Reply:
densmap is installed with umap-learn